AsyncSwift/Views/EventDetailView.swift

```diff
@@ -0,0 +1,110 @@
+//
+//  EventDetailView.swift
+//  AsyncSwift
+//
+//  Created by Kim Insub on 2022/09/08.
+//
+
+import SwiftUI
+
+struct EventDetailView: View {
+
+    private let event: EventModel.Event
+    @StateObject var observed = Observed()
+
+    init(event: EventModel.Event) {
+        self.event = event
+    }
+
+    var body: some View {
+        ScrollView {
+            VStack(alignment: .leading, spacing: 0) {
+                customDivider
+                    .padding(.top, 10)
+                description
+                customDivider
+                information
+                Spacer()
+            }
+        }
+        .navigationTitle(event.detailTitle)
+        .confirmationDialog("", isPresented: $observed.showSheet, titleVisibility: .hidden) {
+            if let naverMapURL = URL(string: event.addressURLs.naverMapURL) {
+                Link("네이버 지도로 길 찾기", destination: naverMapURL)
+            }
+            if let kakaoMapURL = URL(string: "event.addressURLs.kakaoMapURL") {
+                Link("카카오맵으로 길 찾기", destination: kakaoMapURL)
+            }
+        }
+    }
+}
+
+private extension EventDetailView {
+
+    var description: some View {
+        VStack(alignment: .leading) {
+            Text(event.subject)
+                .fontWeight(.bold)
+                .font(.title3)
+            ForEach(event.description, id:\.self) { paragraph in
+                Text(paragraph.content)
+                    .font(.body)
+            }
+            Text(event.hashTags)
+                .padding(.top, 8)
+                .foregroundColor(.gray)
+                .font(.body)
+        }
+        .padding(.horizontal, 24)
+        .padding(.vertical, 30)
+    }
+
+    var information: some View {
+        VStack(alignment: .leading, spacing: 40) {
+            VStack(alignment: .leading, spacing: 8) {
+                Text("\(Image(systemName: "calendar")) Date and time")
+                    .font(.title3)
+                    .fontWeight(.semibold)
+                Text("\(event.date)\n\(event.time)")
+                    .font(.body)
+                Button("캘린더에 추가") {
+                    observed.showingAlert = true
+                }
+                .alert(isPresented: $observed.showingAlert) {
+                    Alert(
+                        title: Text("'AsyncSwift'이(가) 사용자의 캘린터에 접근하려고 합니다."),
+                        message: Text("사용자의 '캘린더'에 접근하도록 허용합니다."),
+                        primaryButton: .default(Text("허용 안 함")) {
+                            observed.showingAlert = false
+                        },
+                        secondaryButton: .default(Text("확인")) {
+                            observed.addEventOnCalendar()
+                        }
+                    )
+                }
+            }
+            VStack(alignment: .leading, spacing: 8) {
+                Text("\(Image(systemName: "location.fill")) Location")
+                    .font(.title3)
+                    .fontWeight(.semibold)
```

Comment:
```suggestion
                    .font(.system(size: 20, weight: .semibold))
```
font weight가 중복 선언되니 한 번만 선언되게끔 하는게 어떤가요?

Reply:
system font 의 type을 사용하지 못하네요 .. ! 

코드 스타일과 상관이 있을 듯한데 다들 어떻게 생각하실까요 ? 👀 @song-cool @insub4067

Reply:
.title3의 가변적 폰트 사이즈에 대응하기 위함이고 
원래 title3의 weight가 regular 이기 때문에 아래에 .semibold로 설정한 내용입니다

폰트 관련 내용의 코드는 디자이너의 요청으로 수정 되어진 내용들입니다

Reply:
@song-cool  관련 내용 추가 설명 필요하면 직접해주시면 좋을거 같습니다~

Reply:
type을 사용할 때와 사용하지 않을 때의 accessibility (기기자체의 텍스트 사이즈) 대응을 비교해보면 어떨까요 ?
이 부분만 System font를 사용하지 않았을 때 사이즈가 가변이지 않는 것에 대한 부자연스러움이 괜찮을까 궁금하네요

Reply:
@unnnyong 동일한 맥락에서 전체적으로 가변폰트를 사용하는게 맞다고 생각했었습니다. 테스트 해보고 결과를 공유해보겠습니다!

부분적으로 고정폰트를 사용해야하는 경우는, 가변시(너무 크거나 작은 폰트를 사용했을 때 더 커지거나 작아져서) 레이아웃에 영향을 주는 경우라고 생각을 합니다. 영의 의견대로 테스트 해보는게 바람직해보입니다! :)